frontend/packages/dev-console/src/components/topology/filters/TopologyFilterBar.tsx

```diff
@@ -71,15 +72,40 @@ const TopologyFilterBar: React.FC<TopologyFilterBarProps> = ({
   const kialiLink = getNamespaceDashboardKialiLink(consoleLinks, namespace);
   const queryParams = useQueryParams();
   const searchQuery = queryParams.get('searchQuery') || '';
+  const [isQuickSearchActive, setIsQuickSearchActive] = React.useState<boolean>(false);
```

Comment:
When I made the comment about moving the button to a separate component, I should have specified that this state should move with it so that it's all encapsulated into a single component.